packages/zowe-explorer/CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Resolved user interface bug with tables that caused an inconsistent table height within the VS Code Panel. [#3389](https://github.com/zowe/zowe-explorer-vscode/pull/3389)
 - Fixed an issue where opening a data set with the same starting pattern as an archived data set caused a REST API error (code 500) to appear in the editor. [#3407](https://github.com/zowe/zowe-explorer-vscode/pull/3407)
 - Fixed an issue where registering new profile types from a Zowe Explorer extender could cause an internal API error on startup. [#3412](https://github.com/zowe/zowe-explorer-vscode/pull/3412)
+- Fixed issue where the 'Delete' key binding for the USS tree returns a 'contextValue' error. [#2796](https://github.com/zowe/zowe-explorer-vscode/issues/2796)
```

Comment:
If this PR is merged before https://github.com/zowe/zowe-explorer-vscode/pull/3421, we may have to move this entry to a new section.
But if https://github.com/zowe/zowe-explorer-vscode/pull/3421 is merged before this one, we will want to move this entry to a new `## TBD Release` section (in this PR) 😢 
Sorry for the inconvenience